eng/common/scripts/get-codeowners.ps1

```diff
@@ -13,34 +16,89 @@ if (!(Test-Path $codeOwnersLocation)) {
 }
 
 $codeOwnersContent = Get-Content $codeOwnersLocation
+$previousLine
```

Comment:
What is this here for?

Reply:
Am using it to capture the the line before the current line in the loop. i.e. (current Loop count -1)

Reply:
You don't have to declare the variable here.